src/Directory.Build.props

```diff
@@ -22,12 +22,18 @@
     <PackageIcon>sentry-nuget.png</PackageIcon>
     <PackageProjectUrl>https://sentry.io</PackageProjectUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
 
     <PackageReleaseNotes>Can be found at: https://github.com/getsentry/sentry-dotnet/releases</PackageReleaseNotes>
 
     <!-- Prop required by Microsoft.SourceLink.GitHub -->
+    <!-- Optional: Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <!-- Optional: Embed source files that are not tracked by the source control manager in the PDB -->
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
What would such files be? This was left out I believe because I didn't understand the value (or couldn't verify it would add value) when I originally added it

Reply:
I added a new comment in the original issue #530 with some screen shots about this. 

If I set that to `true` or `false`, it still says 'untracked'. Hmmm.

When i add the `ContinuousIntegrationBuild` value in .. i can get better results.

`ContinuousIntegrationBuild`added + `false`

![image](https://user-images.githubusercontent.com/899878/96364870-9cdebe80-1188-11eb-9258-1855a3cf72e0.png)

`ContinuousIntegrationBuild` added + `true`

![image](https://user-images.githubusercontent.com/899878/96364888-aff18e80-1188-11eb-8a58-919664496314.png)


but if i don't add the `ContinuousIntegrationBuild` value... 

![image](https://user-images.githubusercontent.com/899878/96364919-e16a5a00-1188-11eb-8f41-4918ad9d0593.png)


---

TL;DR;

To get NuGet package explorer to be 'happy', I need:
- ✔️ `<EmbedUntrackedSources>true</EmbedUntrackedSources>`
- ✔️ `<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>`

I'm honestly not too sure what these are doing and how they make things better/worse. Or in other words, what effect does it have when i DO NOT include them? Sure i know NuGet Package Explorer cries .. but what is happening behind the scenes?

Reply:
Got some info on the [CIB setting](https://github.com/clairernovotny/DeterministicBuilds#deterministic-builds).

> This repo shows how to get a fully deterministic build using Source Link. Deterministic builds are important as they enable verification that the resulting binary was built from the specified source and provides traceability.
>
>Deterministic builds require a property to be set to true during CI: ContinuousIntegrationBuild. These should not be enabled during local dev or the debugger won't be able to find the local source files.

So i'll try and make sure the CIB setting is only set via the CLI (because we're using various CI environments right now)

Reply:
Also, -FINALLY- stumbled across the [docs about the settings](https://github.com/dotnet/sourcelink/blob/master/docs/README.md#embeduntrackedsources).

> Set EmbedUntrackedSources to true to instruct the build system to embed project source files that are not tracked by the source control or imported from a source package to the generated PDB.
>
>If the project generates additional source files that are added to Compile item group in a custom target, this target must run before BeforeCompile target (specify BeforeTargets="BeforeCompile"). Otherwise, these additional source files will not be automatically embedded into the PDB.

Again, I don't really understand what this means with respect to Sentry / Sentry's project system :(

I'm going to leave it IN .. as Nuget Package Manager was having a cry, if it was `false`.